invenio_administration/assets/semantic-ui/js/invenio_administration/src/formik/GenerateForm.js

```diff
@@ -49,6 +59,8 @@ const generateFieldProps = (
     required: fieldSchema.required,
     disabled: fieldSchema.readOnly || (fieldSchema.createOnly && !isCreate),
     placeholder,
+    options: formFieldConfig?.options || fieldSchema?.metadata?.options,
+    rows: formFieldConfig?.rows || fieldSchema?.metadata?.rows,
```

Comment:
```suggestion
    options: formFieldConfig?.options,
    rows: formFieldConfig?.rows,
```

Minor: If I am not wrong the `fieldSchema` is a json representation of the marshmallow schema, this will never have options and rows attributes, unless I am missing something.

Reply:
You're totally right, marshmallow schema doesn't have rows and options, but it has metadata, to which we can nput anything we want. See here: https://github.com/inveniosoftware/invenio-banners/pull/15/files#diff-97da9b93fd82ecb3e421f77c8f0d82f16edeba13ed1b154cdc44572be3c37a3dR25

Reply:
Ah I see, was confused by the naming, we use `metadata` as well for the record's metadata, but I don't have better alternatives, thanks for clarifying!